src/index.tsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+
 import App from './components/App';
 import * as serviceWorker from './serviceWorker';
+import './index.css';
```

Comment:
I've question here:  Is this the only way to define _global_ font for the app by using `index.css` in `index.ts` file? CSS is not flexible format. Have you tried this (to load fonts using JS and expose them in the `theme.ts`)?
![image](https://user-images.githubusercontent.com/14184100/66586916-bcacd100-eb89-11e9-8142-6147c24976b9.png)
From here: https://material-ui.com/customization/typography/

Reply:
Oh sorry @lucianojss. I've read your description. 😀
>I wanted to do it in a more fancy way with webfontloader etc.. but I couldn't manage so I went with the most traditional way with .css file and webpack.
Let me know your thoughts.

If so, maybe I'm gonna look at it later. Let's keep it as it is.
Approved :white_check_mark:

Reply:
Sure @duiliopastorelli